README.md

```diff
@@ -193,3 +193,23 @@ file inside your source code repository.
 
     This variable indicates that the asset compilation process will be skipped. Since this only takes place
     when the application is run in the `production` environment, it should only be used when assets are already compiled.
+
+Hot deploy
+---------------------
+In order to dynamically pick up changes made in your application source code, your application needs to be built with the `rerun` ruby gem. To do so, insert the following line into your Gemfile:
+```
+gem 'rerun', :group => :development
+```
+
+Also your application needs to be either built with [S2I](https://github.com/openshift/source-to-image) passing the `RACK_ENV=development` environment variable, or if the variable is not set in you built image, run [Docker](http://docker.io) with it.
```

Comment:
actually you can scratch that last one for these docs, but we'll want it in the openshift docs for this image.